v1/manifest.go

```diff
@@ -38,10 +39,12 @@ type Descriptor struct {
 	Annotations map[string]string `json:"annotations,omitempty"`
 }
 
-// ParseManifest parses the provided string into a Manifest.
-func ParseManifest(data []byte) (*Manifest, error) {
+// ParseManifest parses the io.ReadCloser's contents into a Manifest.
+func ParseManifest(r io.ReadCloser) (*Manifest, error) {
+	defer r.Close()
+	decoder := json.NewDecoder(r)
```

Comment:
Normally this is just unlined: `json.NewDecoder(r).Decode(&m)`

Reply:
Done in the baseline change.